Website/Packages/datax-pipeline/src/modules/flowDefinition/components/info/infoSettingsContent.jsx

```diff
@@ -73,6 +78,35 @@ export default class InfoSettingsContent extends React.Component {
             </div>
         );
     }
+
+    renderSaveButton() {
+        //Will be enabled when user has access 
+        const enableButton = this.props.saveFlowButtonEnabled;
+        return (
+            <DefaultButton
+                key="save"
+                className="header-button"
+                disabled={!enableButton}
+                title="Save the Flow"
+                onClick={() => this.saveFlowAndInitializeKernel()}
+            >
+                <i
+                    style={enableButton ? IconButtonStyles.greenStyle : IconButtonStyles.disabledStyle}
+                    className="ms-Icon ms-Icon--Save"
+                />
+                Save
+            </DefaultButton>
+        );
+    }
+
+    saveFlowAndInitializeKernel() {
+        Q(
+            this.props.onSaveDefinition()
+        )
+            .then(() => {
+                this.props.getKernel();
+            })
```

Comment:
As we chatted, let's move this to flowDefinitionPanel.

Reply:
Moved to flowDefinitionPanel